README.md

```diff
@@ -33,25 +33,42 @@ annotationProcessor() ...`.
 
 ## Using the checker
 
-There are [separate instructions](README-LOMBOK.md) if your project uses Lombok.
+If your project uses Lombok, also see the further Lombok-specific instructions below.
 
-1. Make your Maven/Gradle project depend on `net.sridharan.objectconstruction:object-construction-checker:0.1.5`.
-
-  For example, for Gradle, add the following to the `build.gradle` file (adding the entries to the extant `repositories` and `dependencies` blocks if present):
+1. Add the [org.checkerframework](https://github.com/kelloggm/checkerframework-gradle-plugin) Gradle plugin to the `plugins` block of your `build.gradle` file:
 
+  ```groovy
+  plugins {
+      ...
+      id "org.checkerframework" version "0.4.0"
```

Comment:
Maybe we should add a note here about using the latest version of the gradle plugin? It is released frequently so it would be good to avoid having `0.4.0` directly in this README.

Reply:
We could change this to `0.4.+` instead of `0.4.0`, so that it will use the latest version in the `0.4.x` series. I don't actually know if that works for plugin versions, but it does for regular gradle versions so I assume so.

Reply:
Unfortunately that doesn't work for plugin versions; dynamic versions are not supported there.  Let me know if you want me to add a note encouraging the user to check for a more recent plugin version.